session/filetransfer.go

```diff
@@ -419,6 +421,30 @@ func (h *FileTransferHandler) FileUploadHandler(
 		done bool
 		open bool
 	)
+	writeChunk := func(msg *ws.ProtoMsg) error {
```

Comment:
is there really a reason for it to not be a regular function? we can have units for it even :)